3604/1/table_output.rb

```diff
@@ -0,0 +1,32 @@
+require 'terminal-table'
+
+# blabla seokvfkse;dofkosekfo
+class TableOutput
+  def table_output(array_gems, count_gems_in_argv)
+    table = Terminal::Table.new
+    table.style = { border_top: false, border_bottom: false }
+    how_many_names_need_to_display?(count_gems_in_argv, array_gems.size).to_i.times do |iter|
+      table.add_row transformation_into_a_nice_view(array_gems[iter])
+    end
+    puts table
+  end
+
+  private
+
+  def transformation_into_a_nice_view(hash)
```

Comment:
please refactor this method
iter = - 1 is a strange line. probably you can just use `each_with_index`
also we can't rely on order of items in hash